src/Castle.Windsor.Tests/TransientMultiConstructorTestCase.cs

```diff
@@ -31,10 +31,10 @@ public void TransientMultiConstructorTest()
 			DefaultKernel container = new DefaultKernel();
 			((IKernel)container).Register(Component.For(typeof(FooBar)).Named("FooBar"));
 
-			var arguments1 = new Dictionary<object, object>();
+			var arguments1 = Arguments.FromDictionary(new Dictionary<object, object>());
 			arguments1.Add("integer", 1);
```

Comment:
This is probably overkill. We could just new up an empty Arguments without using the `FromDictionary` extension.

Reply:
#LetsFixOurTests

Reply:
This could be changed to: 

```csharp
var arguments1 = new Arguments();
```